src/fetch/scripts/setupJest.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+
+import { server } from '../src/test-utils';
+
+beforeAll(() =>
+  server.listen({
+    onUnhandledRequest: 'warn',
```

Comment:
Would `error` or some other kind of hard fail mechanism make more sense here?

Reply:
Yeah, I thought so as well, but it doesn't really work that way. It is just about which console method you want to use, and since we have our test fail on unwanted console logs, this should be good enough.

Reply:
Ah ok. I checked it only briefly and I saw there some callback option as well. So I thought we could throw there. But I didn't spend much time reading it. 😬

Reply:
I thought so as well, and I tried doing so once, but it didn't fail the test.